.github/workflows/unit_test.yml

```diff
@@ -0,0 +1,58 @@
+name: Run-Tests
+
+on:
+  pull_request:
+    branches: [develop, aws-poc]
+
+jobs:
+  scala-style:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - uses: actions/cache@v1
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-
+      - name: Run the scalastyle maven plugin
+        run: mvn -B scalastyle:check
```

Comment:
We have a ton of warnings there. Won't they cause issues?

Reply:
Warnings are ok. I think. It is failing on dependencies